client/src/components/body/Repair/Repair.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-unused-vars */
+import { useState } from "react";
+import Proptypes from "prop-types";
+import GameStateContext from "../../exercise/lab9/Lab9Context";
+import CodeUpdateHeader from "../../exercise/lab3/components/CodeUpdateHeader";
+import React from "react";
+import Button from "../../all-components/Navigation/Button";
+import CodeBlock from "../../all-components/CodeBlock/Codeblock";
+/**
+ * Repair: is a reusable component that is responsible for
+ * allowing for the ability to render and handle new repair pages
+ * moving forward. This is a logic and display component that will use props
+ * to render and display information to the page
+ * @param {Object} props
+ * @returns rendered repair page.s
+ */
+const Repair = (props) => {
+  const {
+    //user = "",
```

Comment:
Is this something we'll need in the future or can we delete this?

Reply:
I can delete I just had it so it would not throw errors @heathermoses